spec/cb/responses/cover_letter/list_spec.rb

```diff
@@ -0,0 +1,79 @@
+# Copyright 2015 CareerBuilder, LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+require 'spec_helper'
+
+module Cb
+  module Responses
+    module CoverLetter
+      describe List do
+        let(:cb_cover_letter_list_response) { described_class.new api_response }
+
+        it { expect(described_class).to be < Cb::Responses::ApiResponse }
+
+        describe '#models' do
+          subject(:cover_letters) { cb_cover_letter_list_response.models }
+
+          context 'when no cover letters are returned' do
+            let(:api_response) { JSON.parse File.read('spec/support/response_stubs/cover_letter/list/none.json') }
```

Comment:
All versions of the APIs support both for GETs and the oldest ones only support json on gets and not posts.